index.html

```diff
@@ -6,27 +6,43 @@
     <link rel="stylesheet" href="index.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
     <link rel="stylesheet" href="./fonts/CreteRound-Italic.ttf">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    
     <style>
       @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;400;700&family=Poppins:wght@100&family=Roboto:wght@100&display=swap');
     </style>
     <title>Mobile Portfolio</title>
 </head>
 <body>
     <header class="header-container">
-      <div class="time-bar">
-        <img class="time-status" src="./final-images/↳ Time.png" alt="">
-        <img class="time-status" src="./final-images/Group 4.png" alt="">
-      </div>
-       <div class="top-navigation">
-          <a href="#">My Logo</a>
-          <nav class="menu-bar">
-            <ul class="desktop-navigation-bar">
-              <li><a href="#">Portfolio</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Contact</a></li>
-              <li><a href="#"><img src="./Desktop-images/Icon - Mail.png" alt=""></a></li>
-            </ul>  
-            <img class="burger-nav" src="./images/Enabled.png" alt="">
+      <div class="time-nav-container">
+        <div class="time-bar">
+          <img class="time-status" src="./final-images/↳ Time.png" alt="">
+          <img class="time-status" src="./final-images/Group 4.png" alt="">
+        </div>
+          <nav class="menu-container">
+            <a class="my-logo" href="#">My Logo</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a class="nav-link" href="#">Portfolio</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">contact</a>
+              </li>
+              <li class="nav-item nav-mail-item">
+                <a class="nav-link" href="#">
+                  <img class="nav-mail-icon" src="./Desktop-images/Icon - Mail.png" alt="">
+                </a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
```

Comment:
You did well with your project, kindly note that each of the menu options should direct you to the right part of the page. That is, the `Portfolio`, `About`, and `Contact` menu option should take you directly to the portfolio,About and Contact section of your page respectively so that a visitor is always directed to the right part of the page 😄